intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2023 James Brine
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+""" IntelMQ parser for jamesbrine.com.au bruteforce IP feed """
+
+import dateutil.parser
+import posixpath
+from urllib.parse import urlparse
+
+from intelmq.lib.bot import ParserBot
+from intelmq.lib.harmonization import DateTime
+
+
+class JamesBrineParserBot(ParserBot):
+    """
+    Parse the jamesbrine.com.au host feed (csv)
+
+    List of source fields:
+    [
+        'ip_address',
+        'activity' (unused),
+        'date'
+    ]
+
+    """
+
+    def parse_line(self, line, report):
+    if line.startswith('#') or len(line) == 0:
+        self.tempdata.append(line)
```

Comment:
`self.tempdata` is never used. Did you intend to write your own `recover_line` method and using that one for `raw`?

Reply:
That was the intention. I've decided instead to adapt the csv endpoint instead such that a customer parser is not required and will be closing this PR.